test/e2e/common/node/container_probe.go

```diff
@@ -257,23 +257,6 @@ var _ = SIGDescribe("Probing container", func() {
 		runReadinessFailTest(ctx, f, pod, time.Minute, true)
 	})
 
-	/*
```

Comment:
O.K., after re-reading a bunch of stuff I wrote 3 years ago I'm confident that we can remove this weird test, yay!

Reply:
:)